src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -51,11 +49,41 @@ class RsErrorAnnotator : Annotator, HighlightRangeExtension {
             override fun visitValueParameter(o: RsValueParameter) = checkValueParameter(holder, o)
             override fun visitVis(o: RsVis) = checkVis(holder, o)
             override fun visitBinaryExpr(o: RsBinaryExpr) = checkBinary(holder, o)
+            override fun visitCallExpr(o: RsCallExpr) = checkCall(holder, o)
```

Comment:
Let's also stick with naming handlers as the corresponding elements are named: `checkCallExpr()` instead of `checkCall()` in this case. It makes searching for visitors and handlers of a particular elements easier.